src/yukon/frontend/src/components/Plotter/Plotter.vue

```diff
@@ -6,7 +6,7 @@
  -->
```

Comment:
We no longer need this component. We talked a lot about Plotter and the related logic but all of that is obviated by the Canvas, so I suggest we drop this component now to avoid confusion.

Reply:
why so? I don't think that the canvas replaces what a plotting tool serves for.

Reply:
It does. I wrote about it in my post https://forum.uavcan.org/t/gui-tool-next-generation/229/53. The data model for plotting was never finalized; I arrived to the concept of canvas while trying to finalize it.

Reply:
I kept the component though it's not accessible through the GUI. We might think of a way of integrate the component on the canvas.